state/state.go

```diff
@@ -3076,14 +3076,18 @@ func (s *State) AddToReplayFilter(mask int, hash [32]byte, timestamp interfaces.
 func (s *State) IsActive(id activations.ActivationType) bool {
 	highestCompletedBlk := s.GetHighestCompletedBlk()
 
-	rval := activations.IsActive(id, int(highestCompletedBlk))
+	active := activations.IsActive(id, int(highestCompletedBlk))
 
-	if rval && !s.reportedActivations[id] {
-		s.LogPrintf("executeMsg", "Activating Feature %s at height %v", id.String(), highestCompletedBlk)
+	if !s.reportedActivations[id] {
+		if active {
+			s.LogPrintf("executeMsg", "Activating Feature %s at height %v", id.String(), highestCompletedBlk)
```

Comment:
isn't highestCompletedBlk  an integer? If so should the string interpolation use %d?

Reply:
%v is the same thing as %d for int, I'll change it for clarity